variables.tf

```diff
@@ -124,6 +124,12 @@ variable "apiary_customer_accounts" {
   type        = "list"
 }
 
+variable "apiary_assume_roles" {
+  description = "Cross account AWS IAM roles allowed write access to managed Apiary S3 buckets using assume policy."
+  type        = "list"
+  default     = []
+}
+
```

Comment:
Since you made this a different variable than `producer_iamroles`, is your intention that people can configure cross-account writes that either do `bucket-access-full-control` *or* do the `assume role`?  It doesn't seem like there's much of a use case for not assuming the role, except for backwards compat.

It almost seems to me like this new list should be part of the `producer_iamroles` variable, and if we need to differentiate between role assumption or not, have a boolean in the map that says whether or not to assume the role.